docs/data-sources/application.md

```diff
@@ -39,6 +39,7 @@ data "qovery_application" "my_application" {
 - `environment_variable_overrides` (Attributes Set) List of environment variable overrides linked to this application. (see [below for nested schema](#nestedatt--environment_variable_overrides))
 - `environment_variables` (Attributes Set) List of environment variables linked to this application. (see [below for nested schema](#nestedatt--environment_variables))
 - `healthchecks` (Attributes) Configuration for the healthchecks that are going to be executed against your service (see [below for nested schema](#nestedatt--healthchecks))
+- `labels_group_ids` (Set of String) list of labels group ids
```

Comment:
To be consistent with existing doc (same for others `.md` files)
```suggestion
- `labels_group_ids` (Set of String) List of labels group ids
```

Reply:
fixed. Thank you